dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WCCustomProperties.java

```diff
@@ -312,6 +312,9 @@
 
     //19.0.0.8
     public static boolean GET_REAL_PATH_RETURNS_QUALIFIED_PATH;
+    
+    //20.0.0.7
+    public static boolean DECODE_URL_PLUS_SIGN;
```

Comment:
These are probably significant enough changes to warrant updating the copyright date range. Thoughts?

Reply:
I updated it for this file.